recipe/build.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 
-./configure  --prefix=$PREFIX
-make -j
+set -xe
+
+# update config.guess and config.sub to newer versions which support more CPU architectures
+rm -f config.sub config.guess
+wget "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD" -O config.guess
+wget "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD" -O config.sub
```

Comment:
Better form would be to require `gnuconfig` and copy from there to replace the build source. There's an example in https://github.com/conda-forge/r-fs-feedstock/pull/27.